### DISCUSSION TITLE
How to increase overlay filesystem size

### DISCUSSION THREAD

**Discussion by Sandmania** [rancher-sandbox/rancher-desktop#2391] [Q&A] [answered]
Hi,

I'm running rancher desktop 1.4.1 on macOS. And for the love of me I can not figure out how I can increase the docker overlay filesystem size from 100GB. Could somebody nudge me to the right direction?

**Comment by jandubois** [comment, marked as answer]
You could try [this](https://github.com/rancher-sandbox/rancher-desktop/issues/1521#issuecomment-1030886234), but it requires you deleting and recreating your current image.

**Reply by Sandmania** [reply]
Thank you for the quick reply.

It kinda works. However not through the `override.yaml`.

I do see log messages like these    :

` time="2022-06-15T11:28:10+03:00" level=debug msg="Mixing \"/Users/user/Library/Application Support/rancher-desktop/lima/_config/override.yaml\" into \"/Users/user/Library/Application Support/rancher-desktop/lima/0/lima.yaml\""`    

In `~/Library/Logs/rancher-desktop/lima.log` 

But there is no `disk` entry in the created `lima.yaml`   
```
❯ grep disk ~/Library/Application\ Support/rancher-desktop/lima/0/lima.yaml | wc -l
       0
```

However, if I directly edit  `~/Library/Application Support/rancher-desktop/lima/0/lima.yaml`. And add the disk entry there, then it seems to work. The setting seems to stick between Rancher Deskop restarts, but can I expect it to get lost at some point? When is the lima.yaml recreated?

**Reply by jandubois** [reply]
You should not edit `lima.yaml`; it can be modified by Rancher Desktop at any time. The content of `override.yaml` will be combined with `lima.yaml` when the VM is created and every time it starts up. The `override.yaml` values are not copied over into `lima.yaml`, but should still take effect.

This is the whole point of `override.yaml` to have a place to override settings that Rancher Desktop will not touch. The only exception is "Factory Reset" in which case the file is deleted with the rest of the directory structure.

**Reply by Sandmania** [reply]
Thank you for the clarification. It does indeed work.
